src/CloudEvents/Event.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CloudEvents;
+
+class Event
+{
+    public const VERSION_1_0 = '1.0';
+
```

Comment:
Would be nice if the class had private fields.

Reply:
Agreed — I'll make that change.